crates/core/component/ibc/src/component/rpc/consensus_query.rs

```diff
@@ -130,6 +132,7 @@ impl<HI: HostInterface + Send + Sync + 'static> ConsensusQuery for IbcQuery<HI>
                 channel_id: chan_id,
                 port_id: PortId::transfer(),
                 channel_end: channel,
+                upgrade_sequence: 0,
```

Comment:
New field; I did not know what to do with it, so opted for the default value (as per protobuf convention).